src/stories/Quiz/Component/QuestionTypes/CoverQuestion.stories.tsx

```diff
@@ -1,48 +1,49 @@
-import { expect } from '@storybook/jest';
-import { CoverQuestion } from '@constructor-io/constructorio-client-javascript/lib/types';
 import type { Meta, StoryObj } from '@storybook/react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
We have `import/no-extraneous-dependencies` in our eslint, why do we need it here?